apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.php

```diff
@@ -74,7 +74,7 @@ function activate_error_reporting() {
  * sticker applied to it.
  */
 function blog_has_error_reporting_sticker() {
-	return has_blog_sticker( 'has-error-reporting', get_current_blog_id() );
+	return \has_blog_sticker( 'has-error-reporting', get_current_blog_id() );
```

Comment:
Strangely enough, the previous namespace-scoped call worked in my manual tests, but was caught in the testing env, here: https://github.com/Automattic/wp-calypso/pull/49485/checks?check_run_id=2450150414.